assets/components/ReportCardContainer.js

```diff
@@ -7,6 +7,8 @@ class ReportCardContainer {
     this.category = data.category;
     this.hazard = data.hazard;
     this.location = data.location;
```

Comment:
what's the difference between data.location and {data.lat and data.lng}?

Reply:
data.location only has the address `text`, data.lat and data.lng has the latitude and longitudes